src/main/java/org/eolang/jeo/representation/directives/DirectivesProgram.java

```diff
@@ -117,14 +117,73 @@ public Iterator<Directive> iterator() {
             .add("listing")
             .set(this.listing)
             .up()
-            .add("errors").up()
-            .add("sheets").up()
-            .add("license").up()
+            .add("errors")
+            .add("error").attr("severity", "warning").set(DirectivesProgram.help()).up()
+            .up()
+            .add("sheets")
+            .add("sheet").set(DirectivesProgram.help()).up()
+            .up()
+            .add("comments")
```

Comment:
@volodya-lombrozo just don't add `comments` at all and you'll be fine. No need to create this fake data.